modules/conversion-center/base/widget-contact-button-base.php

```diff
@@ -582,6 +648,176 @@ private function add_style_top_bar_section(): void {
 		$this->end_controls_section();
 	}
 
+	private function add_style_message_bubble_section(): void {
+		$this->start_controls_section(
+			'style_bubble_section',
+			[
+				'label' => esc_html__( 'Message Bubble', 'elementor' ),
+				'tab'   => Controls_Manager::TAB_STYLE,
+			]
+		);
+	
+		$this->add_control(
+			'style_bubble_colors',
+			[
+				'label'   => esc_html__( 'Colors', 'elementor' ),
+				'type'    => Controls_Manager::SELECT,
+				'default' => 'default',
+				'options' => [
+					'default' => esc_html__( 'Default', 'elementor' ),
+					'custom'  => esc_html__( 'Custom', 'elementor' ),
+				],
+			]
+		);
+	
+		$this->add_control(
+			'style_bubble_name_heading',
+			[
+				'label'     => esc_html__( 'Name', 'elementor' ),
+				'type'      => Controls_Manager::HEADING,
+				'separator' => false,
+			]
+		);
+	
+		$this->add_control(
+			'style_bubble_name_color',
+			[
+				'label'     => esc_html__( 'Text Color', 'elementor' ),
+				'type'      => Controls_Manager::COLOR,
+				'selectors' => [
+					'{{WRAPPER}} .e-contact-buttons' => '--e-contact-buttons-message-bubble-name: {{VALUE}}',
```

Comment:
```suggestion
					'{{WRAPPER}}' => '--e-contact-buttons-message-bubble-name: {{VALUE}}',
```
Do we need the classes in the selectors?

Reply:
Yes, I think we need them because the variables are scoped to the `.e-contact-buttons` class. I tried to remove them and it didn't pick up the change.